### ISSUE TITLE
Can't run Windows Terminal without Administrator

### ISSUE THREAD

**Issue by LoganDark** [microsoft/terminal#13197]
### Windows Terminal version

1.13.11431.0

### Windows build number

10.0.19041.572

Yes, this has actually downgraded since my last bug report. Blame the fact that I had limited download options.

### Steps to reproduce

See https://github.com/microsoft/terminal/issues/13099#issuecomment-1138459416 and the comments leading up to it.

1. I was given a command that completely ruined every app on my system.
2. I had to do an in-place upgrade to fix the issue.
3. Windows Terminal, through any amount of reinstalls, now cannot run without Administrator.

### Actual Behavior

Windows Terminal, through any amount of reinstalls, now cannot run without Administrator. I also cannot double-click the msixbundle to install (I get some permission error), but `Add-AppxPackage` works.

The `wt` command is not found in a non-Administrator Command Prompt, but it is found in an Administrator Command Prompt.

No Start Menu entries work for any part of Windows Terminal, not even the shortcuts, *unless* run as Administrator.

**Comment by lonnywong** [comment]
I reset UAC and reboot, and it works for me.

**Comment by LoganDark** [comment]
> I reset UAC and reboot, and it works for me.

What do you mean "reset UAC"? 

Do you mean turning this slider to the very bottom?

![image](https://user-images.githubusercontent.com/4723091/172904967-9234f47c-3985-4a05-bef3-cb1e1be16242.png)

**Comment by lonnywong** [comment]
"reset" means I didn't change the level.

Mine is "Never notify", I click it, and then click "OK", and then it said I need to reboot. It works after reboot.

Btw, I change the level to default, and reboot again, it works too.

**Comment by lonnywong** [comment]
I'm using the default level, and it works.

I think you can set it to any level, just set it again.

**Comment by LoganDark** [comment]
> 3\. You should getting "restart the computer".

I'm actually not for some reason? I can change the level all I want without restarting. Even completely disable it and re-enable it again and the changes apply instantly (i.e. it's not asking me to reboot). Should I just reboot anyway after messing with it?

**Comment by lonnywong** [comment]
It must be a bug.
I can't reproduce it now. Maybe you should try to reboot anyway.

**Comment by LoganDark** [comment]
Rebooted my computer again, still can't open Windows Terminal without administrator. :/

**Comment by lonnywong** [comment]
Sorry for not helping.
I have the same issue before, and happened to solve it yesterday.

**Comment by zadjii-msft** [comment]
\<meta discussion>

We need a better way of reporting these issues to the appx team - We simply don't have the debugging knowhow to investigate these properly.

**Comment by Trugamr** [comment]
Make sure in `settings.json`, profile settings don't have `elevate: true` setting.

**Comment by LoganDark** [comment]
> Make sure in `settings.json`, profile settings don't have `elevate: true` setting.

There is no `elevate` anywhere in my settings.json

**Comment by LoganDark** [comment]
woah, I have no idea how that got clicked, I used ctrl+enter to comment

**Comment by JonathanPitre** [comment]
Same issue here when I run Terminal (Admin) with a non-admin account and input my admin creds.
Even with the Terminal Preview version,

Is this being looked into? This a fresh build of Windows 11 22H2.

**Comment by LoganDark** [comment]
Okay, does *this* have something to do with it?

![image](https://user-images.githubusercontent.com/4723091/199488633-20e147b7-8c3d-45ff-9921-151728821321.png)

I guess this must've happened when I reinstalled Windows. Except... from an admin command prompt.. it is still 0 bytes and yet executes just fine

![image](https://user-images.githubusercontent.com/4723091/199488824-42117c28-ee67-4540-982d-12fc3e4cd041.png)

Huh?!

**Comment by zadjii-msft** [comment]
@LoganDark Nope, those are App Execution Aliases, which are reparse points, which the OS does a really bad job of telling you what they mean. `pwsh` is at least a _little_ better

**Comment by LoganDark** [comment]
> @LoganDark Nope, those are App Execution Aliases, which are reparse points, which the OS does a really bad job of telling you what they mean. `pwsh` is at least a _little_ better

Is there any way to figure out why non-administrators cannot execute them?

**Comment by zadjii-msft** [comment]
> why non-administrators cannot execute them?

I mean, the [reparse point](https://learn.microsoft.com/en-us/windows/win32/fileio/reparse-points) isn't an executable itself. It's just like, a symlink, that points at the actual exe. Trick is, that AEA's for Store apps point at an exe on the filesystem that has super restricted permissions, such that even admins can't execute it normally. However, there's a magic handoff in CreateProcess that knows how to delegate the creation of a packaged process to the right set of ACLs. It's a process I don't fully understand, to be totally honest. That magic lets apps run with special store-granted permissions only, but from the user context it seems like nothing weird happened

It's definitely weird, though, that you can't run the Terminal without admin permissions. That's unexpected. Does that repro for any other Store apps? Like, maybe `Ubuntu.exe`? (I'm sure there are plenty of other Store apps that are "centennial" apps that request the normal level of permissions)

**Comment by LoganDark** [comment]
> I mean, the [reparse point](https://learn.microsoft.com/en-us/windows/win32/fileio/reparse-points) isn't an executable itself. It's just like, a symlink, that points at the actual exe.

Yeah, I understand what you mean. You still *execute* them in order to do the reparse magic, though.

> It's definitely weird, though, that you can't run the Terminal without admin permissions. That's unexpected. Does that repro for any other Store apps? Like, maybe `Ubuntu.exe`? (I'm sure there are plenty of other Store apps that are "centennial" apps that request the normal level of permissions)

The `python.exe` standin that opens the Microsoft Store also only works from an admin command prompt. App Installer also does not work without being granted admin, so for example, I can't double-click appxbundles to install, I have to use `Add-AppxPackage`.

However, *since the in-place reinstall*, built-in apps like Alarms & Clock or Calendar work just fine.

**Comment by zadjii-msft** [comment]
> The `python.exe` standin that opens the Microsoft Store also only works from an admin command prompt ... built-in apps like Alarms & Clock or Calendar work just fine.

Okay yea this is definitely a Centennial issue then. Alarms&Clock, Calendar, Calculator - these are all apps that run in an app container. That's fundamentally different than Terminal, Python, WSL distros, etc, which do not have an app container. 

We need to connect you up with someone on the appx/msix/deployment team to help investigate this, cause this is definitely not something _we_ (the Terminal team) are equipped to debug.

**Comment by LoganDark** [comment]
> > The `python.exe` standin that opens the Microsoft Store also only works from an admin command prompt ... built-in apps like Alarms & Clock or Calendar work just fine.
> 
> Okay yea this is definitely a Centennial issue then. Alarms&Clock, Calendar, Calculator - these are all apps that run in an app container. That's fundamentally different than Terminal, Python, WSL distros, etc, which do not have an app container.

So far I haven't had any issues with accessing WSL, either through Windows Terminal (as admin), `bash.exe` (WSL 2), or `ubuntu2004.exe` (WSL 1).

> We need to connect you up with someone on the appx/msix/deployment team to help investigate this, cause this is definitely not something _we_ (the Terminal team) are equipped to debug.

I would definitely appreciate that very much, thank you! My email can be found on my GitHub profile. You're welcome to have someone contact me, and if we can find a solution I'll make sure to post it here.

**Comment by maidamai0** [comment]
> > Make sure in `settings.json`, profile settings don't have `elevate: true` setting.
> 
> There is no `elevate` anywhere in my settings.json

there are two `settings`, one for `user` and one for `default`, do you check all of them?

**Comment by LoganDark** [comment]
> > > Make sure in `settings.json`, profile settings don't have `elevate: true` setting.
> > 
> > 
> > There is no `elevate` anywhere in my settings.json
> 
> there are two `settings`, one for `user` and one for `default`, do you check all of them?

This isn't the issue because running wt.exe directly raises this error

![image](https://user-images.githubusercontent.com/4723091/200127593-60bfba0f-6311-4bd7-8fc0-5639cf0f655a.png)

**Comment by spufidoo** [comment]
I have the same issue - I cannot start Windows Terminal outside of Administrator Mode, whether it's 
Window Terminal Version 1.18.1462.0, or Windows Terminal Preview Version: 1.18.1462.0, which is a real pain.

**Comment by ttxs69** [comment]
> 1. open the UAC.
> 2. click the level. No matter which level you are using, and you don't need to change it, just click it.
> 3. Click OK. You should getting "restart the computer".
> 4. Reboot the computer. It should work.

This solves my problem, thanks a lot.

**Comment by MeikelLP** [comment]
Still happening to me on a fresh install of win11.
Installed via winget (as admin)

**Comment by rifkiamil** [comment]
## Bug: Windows Terminal (wt.exe) Always Prompts for Administrator Privileges


When launching Windows Terminal (wt.exe), it consistently prompts for administrator privileges (User Account Control dialog appears). 

```
wt.exe
whoami /groups
Group Name: Mandatory Label\High Mandatory Level
```

However, when running cmd.exe or powershell.exe directly, no elevation is required.
Open Command Prompt or PowerShell as a normal user (no elevation):

```
whoami /groups
Group Name: Mandatory Label\Medium Mandatory Level
```

### Other Test
I tried creating a manual shortcut on the desktop pointing to:
```
%LocalAppData%\Microsoft\WindowsApps\wt.exe
```
Launching from this shortcut still prompts for UAC administrator elevation, confirming that the behavior persists regardless of how wt.exe is executed.


## User Account Control Dialogue
```
User Account Control
Do you want to allow this app to make changes to your device?

Terminal

Verified publisher: Microsoft Corporation

File origin: Hard drive on this computer

Program location:
"C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_1.22.11141.0_x64__8wekyb3d8bbwe\WindowsTerminal.exe" --profile "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}" --startingDirectory "C:\Users\r"

[Change when these notifications appear]
[Hide details]

[Yes] [No]
```

**Comment by DHowett** [comment]
@rifkiamil Can you share your `settings.json` file? It is present at `%LOCALAPPDATA%\Packages\Microsoft.WindowsTerminal_8wekyb3d8bbwe\LocalState\settings.json`.

**Comment by rifkiamil** [comment]
@DHowett, I removed the elevate: true line from regarding powershell, which was set as the default for wt.exe, and it looks like that solved the problem. Any idea what application or Visual Studio extension might have had access to do this, or if it could be controlled via Group Policy? Thanks for your help, I did find this setting page before by pres ctrl+shift+, which opened up the json in vscode. 

![Image](https://github.com/user-attachments/assets/36863694-e9e3-417b-896d-516928c35a04)

**Comment by Gamer069** [comment]
> Okay, does _this_ have something to do with it?
> 
> ![image](https://user-images.githubusercontent.com/4723091/199488633-20e147b7-8c3d-45ff-9921-151728821321.png)
> 
> I guess this must've happened when I reinstalled Windows. Except... from an admin command prompt.. it is still 0 bytes and yet executes just fine
> 
> ![image](https://user-images.githubusercontent.com/4723091/199488824-42117c28-ee67-4540-982d-12fc3e4cd041.png)
> 
> Huh?!
> 
> ![image](https://user-images.githubusercontent.com/4723091/199489211-2a155b9d-cea9-497c-a3b6-7a144eb24972.png)

i don't think that matters.
Its showing up as 0KB probably bc its a symlink to C:\Program Files\WindowsApps\ and explorer is sometimes buggy with symlinks.

**Comment by mangodev01** [comment]
i'm facing the same issue. Has anyone been able to figure out the fix for it? (i'm aware that its more of an issue with appx/msix or whatever but still)

**Comment by mangodev01** [comment]
and for some reason if there is an admin session of windows terminal, all non-admin launches work, it's just that if there's no admin window prior to the non-admin launch, "Access is denied."
